src/dynamic_honey_badger/batch.rs

```diff
@@ -119,4 +124,12 @@ impl<C, N: NodeIdT> Batch<C, N> {
             && self.netinfo.public_key_map() == other.netinfo.public_key_map()
             && self.encryption_schedule == other.encryption_schedule
     }
+
+    /// Returns the signature that can be used as a pseudorandom value: None of the validators knew
```

Comment:
Please correct the last sentence to make it more readable. Did you mean "None of the validators of the batch knew this signature before other validators decided."?

It reads like the first sentence might end with a period instead of the colon.

Reply:
Ah, I was meaning to write "before the other _fields_ of the batch were decided". I'll try to come up with a better formulation anyway…